### DISCUSSION TITLE
Custom Theme

### DISCUSSION THREAD

**Discussion by tigawanna** [Tencent/cherry-markdown#631] [Q&A] [answered]
Are there any docs on the theme config and the available options? , Is there a way to toggle the CSS values (data-theme) to customize the colors

**Comment by RSS1102** [comment, marked as answer]
You can refer to here and copy CSS to [create your own theme](https://github.com/Tencent/cherry-markdown/blob/main/src/sass/themes/blue.scss) 

[class name](https://github.com/Tencent/cherry-markdown/blob/b64aeb82f2f6b171546ebcddad03938aef91faca/src/sass/themes/blue.scss#L56C7-L56C7)

```css
.cherry.theme__ (you want to class name){
}
```

``` diff
  /**
   * 配置主题，第三方可以自行扩展主题
   */
  theme: [
    { className: 'default', label: '默认' },
    { className: 'dark', label: '暗黑' },
    { className: 'light', label: '明亮' },
    { className: 'green', label: '清新' },
    { className: 'red', label: '热情' },
    { className: 'violet', label: '淡雅' },
    { className: 'blue', label: '清幽' },
+ { className: 'you want to class name', label: 'you want to name' },
  ],
  ```

**Reply by tigawanna** [reply]
i see, how do i toggle between themes in the config?

**Reply by RSS1102** [reply]
The `theme` JSON has clearly shown you how to add configurations, 
and the rest is to click here：
![image](https://github.com/Tencent/cherry-markdown/assets/81673017/17a49ae9-4da0-4ed0-ae12-a9c464890dac)

**Comment by sunsonliu** [comment]
https://github.com/Tencent/cherry-markdown/wiki/%E9%85%8D%E7%BD%AE%E4%B8%BB%E9%A2%98
The theme configuration has been updated to the wiki
